CONTRIBUTING.md

```
@@ -21,40 +14,36 @@ rustup override set 1.85.0
    git clone https://github.com/<your-username>/statespace.git
    cd statespace
    ```
-2. Build:
+2. Install dependencies:
    ```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)